__tests__/app.test.js

```diff
@@ -287,3 +287,59 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("allows comment to be posted on article with valid ID", () => {
+    const testComment = { username: "icellusedkars", comment: "I agree!" };
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(testComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toEqual(
+          expect.objectContaining({
+            comment_id: 19,
+            body: "I agree!",
+            article_id: 5,
+            author: "icellusedkars",
+            votes: 0,
```

Comment:
check created_at exists too - you don't have to use the time stamp, simply checking it's a valid string is fine